src/pages/index.tsx

```diff
@@ -41,7 +43,7 @@ function HomePage() {
           </div>
           <div>
             <h4 className={styles["modal-wrapper__status"]}>Current Status</h4>
-            <DropDown
+            <Dropdown
               options={[
                 { value: "todo", label: "Todo" },
```

Comment:
It's better to use options like: ['todo', 'doing', 'done'];
Even better to create utils/constant.ts and define them as COLUMNS

Reply:
ok but we will be using data model to determine these options because you can have multiple columns called different things totally. What do you think?